ocs_ci/deployment/tnf.py

```
@@ -0,0 +1,308 @@
+        logger.info("OCP deployment completed")
+
+
+class TNFDeployment(Deployment):
+    """
+    Two-Node Failover OpenShift Data Foundation Deployment class
+
+    This class handles the complete deployment of ODF on two-node clusters
+    including DRBD configuration for the floating monitor.
+    """
+
+    def __init__(self):
```

Ruff (0.15.20)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)